js/src/personal/maliha/Maliha.module.css

```diff
@@ -1,72 +1,121 @@
 .title {
-  color: #b87171;
-  position: relative;
-  font-size: 125px;
-  font-family: "Playfair Display", Serif;
+  position: absolute;
+  font-size: 450px;
+  font-family: "Yeseva One", Serif;
+  letter-spacing: 6px;
+  top: 200px;
+  left: 350px;
+  font-weight: 999;
+}
+
+#title1 {
```

Comment:
nit: prefer classes if non-specific (eg is title1 like header1, header2, etc. that represent some info / page hierarchy?) or name these id's something slightly more specific (eg about_title_1, about_section_title_1)

Reply:
named them more specific things